core-java-modules/core-java-lang-operators/pom.xml

```diff
@@ -17,6 +17,12 @@
     </parent>
 
     <dependencies>
+    	<dependency>
```

Comment:
```suggestion
    	  <dependency>
```

Reply:
just indent this line two more spaces

Reply:
My bad. There was a tab instead of four spaces.